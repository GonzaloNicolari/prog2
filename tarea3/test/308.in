# prueba cadena_a_binario
cadena_a_binario .
cadena_a_binario (4,a) .
cadena_a_binario (4,a) (2,b) (3,c) .
cadena_a_binario (4,a) (2,b) (3,c) (7,d) .
cadena_a_binario (4,a) (2,b) (3,c) (1,d) (5,e) (1,f) (7,g) (1,h) (4,i) (2,j) (3,k) (1,l) (5,m) (1,n) (7,o) .
cadena_a_binario (4,a) (2,b) (3,c) (1,d) (5,e) (1,f) (7,g) (1,h) (4,i) (2,j) (3,k) (1,l) (5,m) (1,n) (7,o) (2,p) .
Fin
