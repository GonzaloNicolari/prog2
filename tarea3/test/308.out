1>#  prueba cadena_a_binario.
2>
3>
(4,a)
4>
-(3,c)
(2,b)
-(4,a)
5>
-(7,d)
(3,c)
-(2,b)
--(4,a)
6>
---(7,o)
--(1,n)
---(5,m)
-(1,l)
---(3,k)
--(2,j)
---(4,i)
(1,h)
---(7,g)
--(1,f)
---(5,e)
-(1,d)
---(3,c)
--(2,b)
---(4,a)
7>
---(2,p)
--(7,o)
---(1,n)
-(5,m)
---(1,l)
--(3,k)
---(2,j)
(4,i)
---(1,h)
--(7,g)
---(1,f)
-(5,e)
---(1,d)
--(3,c)
---(2,b)
----(4,a)
8>Fin.
